### PR TITLE
Render documents using npm command

### PR DESCRIPTION
**Pull request by Rafalkufel**
* `fixtures.js` splitted into common fixture and specifcfixtures. The latter contains data specific to document type.
* fixed document rendering using npm command,
* stub for testing rendering documents

### CODE REVIEW

File: src/docx/documentRender.test.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
+const path = require("path");
+
+const { generateDocument } = require("./generateDocument");
+const { getSpecificData } = require("./specificData");
+const { DocumentData } = require("../utils/utils");
+
+describe("In SPECJALNE decision", () => {
+  const issue = "SPECJALNE";
+  var specificData;
```
Comment (Maciek246) [inline src/docx/documentRender.test.js:9]: `var` should be replaced by `let` keyword from ES5. Here and in other tests.

File: src/docx/documentRender.test.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  beforeEach(() => {
+    specificData = getSpecificData(issue);
+  });
+
+  it("child name should be rendered properly", () => {
+    specificData.child.name = "mock child name";
+    const documentData = new DocumentData(specificData).templateData;
+
+    return generateDocument(issue, documentData)
```
Comment (Maciek246) [inline src/docx/documentRender.test.js:18]: Is`return` necessary?

File: src/docx/testGenerateDocument.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const fs = require("fs");
```
Comment (Maciek246) [inline src/docx/testGenerateDocument.js:1]: Please change the name on more meaningful (which describe what this script do)

File: src/docx/documentRender.test.js (hunk 2 above)
Comment (Rafalkufel) [reply src/docx/documentRender.test.js:18]: See: https://jestjs.io/docs/en/asynchronous#promises

**Pull request merged**